src/briefcase/exceptions.py

```diff
@@ -156,7 +156,13 @@ def __init__(self, platform):
 class InvalidSupportPackage(BriefcaseCommandError):
     def __init__(self, filename):
         self.filename = filename
-        super().__init__(f"Unable to unpack support package {filename!r}")
+        super().__init__(f"Unable to unpack support package '{filename}'.")
```

Comment:
If you just call `repr()` here and `filename` is a `Path`, it looks like this:

```
Unable to unpack support package PosixPath("/asdf")
```
so, this strategy gets what we actually want.
```
Unable to unpack support package '/asdf'
```

Reply:
Torn between whether we should use the explicit `'`, or `{str(filename)!r}` - that should ensure that any inline quotes are correctly escaped (I think...)

Reply:
You mean like these differences?
```pycon
>>> file_path=Path("/pa't'h/to/file")
>>> 
>>> print(f"File is {file_path}")
File is /pa't'h/to/file
>>> 
>>> print(f"File is {str(file_path)!r}")
File is "/pa't'h/to/file"
>>> 
>>> print(f"File is '{file_path}'")
File is '/pa't'h/to/file'
```
Manually calling `str()` does seem to handle that edge case best.

Reply:
That's what I had in mind, yes.